app/controllers/collections_controller.rb

```diff
@@ -11,52 +11,71 @@ def new
     collection = Collection.new(creator: current_user)
     authorize! collection
 
-    @form = CollectionForm.new(collection)
-    @form.prepopulate!
-  end
-
-  def edit
-    collection = Collection.find(params[:id])
-    authorize! collection
-
-    @form = CollectionForm.new(collection)
+    collection_version = CollectionVersion.new(collection: collection)
+    @form = CreateCollectionForm.new(collection_version: collection_version, collection: collection)
     @form.prepopulate!
   end
 
   def create
     collection = Collection.new(creator: current_user)
+
     authorize! collection
 
-    @form = collection_form(collection)
+    collection_version = CollectionVersion.new(collection: collection)
+    @form = collection_form(collection_version)
     if @form.validate(collection_params) && @form.save
-      after_save(collection)
+      after_save(collection: collection, collection_version: collection_version)
     else
       render :new, status: :unprocessable_entity
     end
   end
 
+  def edit
+    collection = Collection.find(params[:id])
+    collection_version = collection.head
+    authorize! collection_version
```

Comment:
When running locally with `REMOTE_USER=auser@stanford.edu ROLES=dlss:hydrus-app-administrators bin/rails server` current_user is failing the `authorize!` call and redirecting to the dashboard when trying to edit a Collection. Is there another role I should add?

Reply:
I fixed this by tweaking the CollectionVersionPolicy to checking for the object to be `updatable?` rather than `can_update_metadata?`